manifests/params.pp

```diff
@@ -11,6 +11,7 @@
   $package_name = 'rundeck'
   $package_ensure = 'installed'
   $service_name = 'rundeckd'
+  $service_restart = true
```

Comment:
can you please assign the value directly in the init.pp? This allows us to get rid of the params.pp in the future and enables puppet-strings to pickup the value for the REFERENCE.md.

Reply:
ok.